### ISSUE TITLE
Crash when given the '--lib' sub command for `new` 

### ISSUE THREAD

**Issue by cowboy8625** [cowboy8625/pygo#2]
I get this panic even though `pygo` does create everything.
```
thread 'main' panicked at 'Could not create Poject Folder.: Os { code: 17, kind: AlreadyExists, message: "File exists" }', /home/cowboy/.cargo/registry/src/github.com-1ecc6299db9ec823/pygo-0.1.0/src/main.rs:80:22
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by cowboy8625** [comment]
fix with [commit](fc4858d932ceef81faa74415f75bebd70ef1788f)

**Comment by cowboy8625** [comment]
close
